### PR TITLE
Fix/title length over 50 symbols error. Page content could be empty on save now. Title could not be empty on save 

### PR DESCRIPTION
**Pull request by ovchynnikov**

### CODE REVIEW

File: frontend/src/components/pages/components/content-editor/content-editor.tsx [hunk 1]
```diff
@@ -52,16 +52,30 @@ export const ContentEditor: React.FC = () => {
...
+          pagesActions.editPageContent({
+            pageId: paramsId,
+            title: titleInputValue.trim(),
+            content: markDownContent?.length === 0 ? ' ' : markDownContent,
+          }),
+        );
+        history.push(replaceIdParam(AppRoute.PAGE, paramsId || ''));
+      }
+    } else if (titleInputValue?.trim().length === 0) {
+      toast.warning('Sorry, title could not be empty');
```
Comment (AndriiPopov3) [inline frontend/src/components/pages/components/content-editor/content-editor.tsx]:
```suggestion
      toast.error('Error: Page title cannot be empty');
```
"Sorry" sounds too unfitting for an error message

File: frontend/src/components/pages/components/content-editor/content-editor.tsx [hunk 2]
```diff
@@ -52,16 +52,30 @@ export const ContentEditor: React.FC = () => {
...
+    } else {
+      toast.warning(
+        `Sorry, title could not be so long. Please delete ${
```
Comment (AndriiPopov3) [inline frontend/src/components/pages/components/content-editor/content-editor.tsx]: Same here

File: frontend/src/components/pages/components/content-editor/content-editor.tsx (hunk 1 above)
Comment (ovchynnikov) [reply frontend/src/components/pages/components/content-editor/content-editor.tsx]: @aleksandr-kuharenko could you please suggest what fits better here? From the UX perspective "Sorry" is always a good idea
Thanks

Comment (ovchynnikov) [reply frontend/src/components/pages/components/content-editor/content-editor.tsx]: a warning has yellow pop-up color, it looks better than a red Error pop-up as I could say

**Review by mykytabatrak — APPROVED** [review]

File: frontend/src/components/pages/components/content-editor/content-editor.tsx (hunk 1 above)
Comment (aleksandr-kuharenko) [reply frontend/src/components/pages/components/content-editor/content-editor.tsx]:
```suggestion
      toast.warning('Title could not be empty');
```

**Review by sofiia-chorna — APPROVED** [review]

File: frontend/src/components/pages/components/content-editor/content-editor.tsx (hunk 2 above)
Comment (ovchynnikov) [reply frontend/src/components/pages/components/content-editor/content-editor.tsx]: Sorry deleted

**Pull request merged**